tests/browsercontext-fetch.spec.ts

```diff
@@ -78,7 +78,7 @@ it('should throw on network error after redirect', async ({ context, server }) =
     req.socket.destroy();
   });
   const error = await context.request.get(server.PREFIX + '/redirect').catch(e => e);
-  expect(error.message).toContain('socket hang up');
+  expect(error.message).toBe('APIRequestContext.get: socket hang up');
```

Comment:
this is inconsistent with our docs where we use `apiRequestContext.get`, let's apply the same logic as in the [docgen](https://github.com/microsoft/playwright/blob/38dcfd785cc1e47bd757d49f6e9ed69859456dc8/utils/doclint/documentation.js#L170) ?